### ISSUE TITLE
Add maximum quantity of toasts that can be displayed at the same time

### ISSUE THREAD

**Issue by deivuss331** [marverix/v-tostini#24]
Hello,

Is there a way to achieve that already? If there's not, I'm wondering what do you think about the idea to add new config option to control the maximum quantity of toasts that can be displayed at the same time?

Please let me know.

**Comment by marverix** [comment]
Hello Dawid,

Nope, there is no such mechanism implemented. It would require implementing some queue mechanism.
The question is: why you need it? Normally you don't want to spam users with a ton of toasts. Ok, but the queue will not help with it - instead, you will introduce another problem - user will see (probably) an infinite train of toasts :)
If you could elaborate on your problem, then we could figure out if you really need a mechanism to "limit" the number of displayed toasts. As I said - maybe a problem is somewhere else.

Br,
Marek

**Comment by deivuss331** [comment]
Hi Marek,

You're absolutely right. I had on my mind the situation to display a toast on form submit in case some fields are not valid and display it only once at the time. What I did is to simply check whether the toast is active (then return) or not (then display toast). It solved my problem which actually wasn't related to v-tostini.

The issue seems to be pointless now.

**Comment by deivuss331** [comment]
Hi Marek,

You're absolutely right. I had on my mind the situation to display a toast on form submit in case some fields are not valid and display it only once at the time. What I did is to simply check whether the toast is active (then return) or not (then display toast). It solved my problem which actually wasn't related to v-tostini.

The issue seems to be pointless now.
Anyway, thank you for your response and help.

Btw. the toast train you mentioned was the mechanism I found in some more popular toasts for vue and yep you're right again, it was a problem ;)
